packages/mui-material/src/CssBaseline/CssBaseline.d.ts

```diff
@@ -6,6 +6,7 @@ export interface CssBaselineProps extends StyledComponentProps<never> {
    * You can wrap a node.
    */
   children?: React.ReactNode;
+  enableColorScheme?: boolean;
```

Comment:
```suggestion
    /*
     * Enable `color-scheme` css property to use `theme.palette.mode`.
     * For more details, check out https://developer.mozilla.org/en-US/docs/Web/CSS/color-scheme
     * For browser support, check out https://caniuse.com/?search=color-scheme
     */
    enableColorScheme?: boolean;
```

Let's add the comment here too